exporters/holstebro_helpers.py

```diff
@@ -0,0 +1,811 @@
+    """ Traverses a tree of OUs, for each OU finds the manager of the OU.
+    : param nodes: The nodes of the OU tree
+    """
+    managerHelper = HolstebroHelper(mh)
+
+    for node in PreOrderIter(nodes['root']):
+        ou = mh.read_ou(node.name)
+        # for each ou, check if name contains _leder
+        # if so, check ou for "tilknytninger" and set this as leader for ou.parent
+        if _is_leader_unit(ou):
+            # We have an Afdeling with name _leder, find associated employees and make them leaders in the parent ou
+            associated_employees = mh.read_organisation_people(
+                node.name, 'association', False)
+
+            # Find the manager if an employee is associated with manager departement
+            manager = {}
+            if len(associated_employees) > 0:
+                # Take the first employees associated with the ou and make them leader
+                manager_uuid = list(associated_employees)[0]
+                manager = associated_employees[manager_uuid]
+
+            if len(associated_employees) > 1:
+                # This is an error. There should be no more than one employeen in manager unit.
+                # Log this as an error.
+                logger.error(
+                    "More than one manager associated with: {}".format(ou['name']))
+
+            if ou['parent'] != None:
+                managerHelper.update_manager(
+                    ou['parent'], manager)
+
+            # This manager is now manager for parent ou - or has been removed
+            # if parent ou's name ends with "led-adm", make employee
+            # manager for the parent ou's parent as well.
+            if _is_cm_unit(ou):
+                managerHelper.update_manager(
+                    ou['parent']['parent'], manager)
+
+
+def _is_leader_unit(ou):
+    is_leader_unit = False
+
+    if ou['org_unit_level']['name'] == 'Afdelings-niveau' and ou['name'].count(SETTINGS['imports.holstebro.leaders.manager_extension']) == 1 and ou['name'].count(SETTINGS['imports.holstebro.leaders.manager_prefix_exclude']) == 0:
+        is_leader_unit = True
+
+    return is_leader_unit
+
+
+def _is_cm_unit(ou):
+    is_cm_unit = False
+
+    if ou['parent']['parent'] != None and ou['parent']['name'].count(SETTINGS['imports.holstebro.leaders.common_management_name']) == 1:
+        is_cm_unit = True
+
+    return is_cm_unit
+
+
+def find_org_manager(mh, node):
+    """
+    Searches for a manager for the given node.
+    Returns the manager directly associated with the node
+    OR the enherited manager from higherlevel
+    If no managers at all in organisation, the empty
+    dict is returned
+    """
+
+    if node == None:
+        return {'uuid': ''}
+
+    new_manager = mh.read_ou_manager(node.name, True)
+
+    if new_manager:
+        return new_manager
+    elif node.depth != 0:
+        return find_org_manager(mh, node.parent)
+    else:
+        return {'uuid': ''}
```

Comment:
Fordi der er tale om retur kode ville jeg skrive det sådan her:
```suggestion
    if new_manager:
        return new_manager
    if node.depth != 0:
        return find_org_manager(mh, node.parent)
    return {'uuid': ''}
```